corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -19,39 +31,57 @@ hqDefine('accounting/js/pricing_table', function () {
         self.selectCurrentPlan = function () {
             self.selected_edition(self.currentEdition);
         };
+        self.capitalizeString = function (s) {
+            return s.charAt(0).toUpperCase() + s.slice(1);
+        };
+        self.isDowngrade = function (oldPlan, newPlan) {
+            if (oldPlan === 'Enterprise') {
+                if (newPlan === 'Advanced' || newPlan === 'Pro' ||
```

Comment:
These conditions aren't terrible but they'd be a bit more readable with [contains](https://underscorejs.org/#contains): `_.contains(['Advanced', 'Pro', 'Standard', 'Community'], newPlan)`